File: src/test/java/uk/gov/justice/hmpps/prison/health/PingEndpointTest.java
package uk.gov.justice.hmpps.prison.health;


import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class PingEndpointTest {
    @Test
    public void ping() {
        assertThat(new PingEndpoint().ping()).isEqualTo("pong");
    }
}

File: src/test/java/uk/gov/justice/hmpps/prison/health/PingEndpointTest.kt
package uk.gov.justice.hmpps.prison.health

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class PingEndpointTest {
  @Test
  fun ping() {
    assertThat(PingEndpoint().ping()).isEqualTo("pong")
  }
}
